# add local .libs directory to library search path during build of zsdltest
--- src/Makefile.am.orig	2018-10-27 19:44:55.332591275 +0200
+++ src/Makefile.am	2018-10-27 19:45:54.268730862 +0200
@@ -70,10 +70,11 @@
 library_includedir = $(includedir)/libzia
 
 zsdltest_SOURCES = zsdltest.c
-zsdltest_LDFLAGS = -lzia 
+zsdltest_LDFLAGS = -lzia -lm
+zsdltest_LDADD = -L.libs
 
-gpiotest_SOURCES = gpiotest.c
-gpiotest_LDFLAGS = -lzia 
+#gpiotest_SOURCES = gpiotest.c
+#gpiotest_LDFLAGS = -lzia 
 
 EXTRA_DIST = regcomp.c regex_internal.c regexec.c 
 
